api/scripts/migrate-audio.ts

```diff
@@ -0,0 +1,109 @@
+import process from "process";
+import { program } from "commander";
+import pkg from "pg";
+const { Client } = pkg;
```

Comment:
Bit of an odd way to import. `import { Client } from "pg";` not possible? not that important though